docs/content/docs/try-flink-ml/java/quick-start.md

```diff
@@ -54,23 +63,17 @@ export FLINK_HOME=`pwd`
     available)
 ### Build Flink ML library
 
-In order to use Flink ML's CLI you need to have the latest binary distribution
-of Flink ML. You can acquire the distribution by building Flink ML's source code
-locally with the following command.
-
-```bash
-cd ${path_to_flink_ml}
-mvn clean package -DskipTests
-cd ./flink-ml-dist/target/flink-ml-*-bin/flink-ml*/
-```
+Please walk through this [guideline]({{< ref "docs/development/building" >}}) to
```

Comment:
Would it be more straightforward to use `docs/development/building/#build-flink-ml-java-sdk` as the reference link?

For released versions, the `build-your-own-project.md` should not require user to build Flink ML library from source. They should be able to download jars directly. Could we use the stable/unstable tags to differentiate these two cases in this README?